cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -45,7 +46,7 @@ ::std_msgs::ColorRGBA ToMessage(const cartographer::io::FloatColor& color) {
 visualization_msgs::Marker CreateTrajectoryMarker(const int trajectory_id,
                                                   const std::string& frame_id) {
   visualization_msgs::Marker marker;
-  marker.ns = "Trajectory " + std::to_string(trajectory_id);
+  marker.ns = absl::StrCat("Trajectory ", std::to_string(trajectory_id));
```

Comment:
you can remove the std::to_string(...) absl::StrCat handles this internally.

Reply:
Cool, didn't know. Will change